.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@
 exclude: '^docs/conf.py'
 
 default_language_version:
-    python: python3.10
```

Comment:
Why is this needed?

Probably not important here, but I wonder if this would cause issues in the official repository. We should dicsuss how we do linting as to avoid conflicts with the official repo in general in our next meetings.